content/kotsadm/registries/kurl-registry.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2019-10-09
+linktitle: "Embedded kURL registry"
+title: Embedded kURL registry
+weight: 10020
+---
+
+Kots can be installed as an add-on on a [kURL](https://kurl.sh/docs/introduction/) cluster, which can also include the [registry add-on](https://kurl.sh/docs/add-ons/registry).
+When installing in [air gap](/kotsadm/installing/installing-embedded-cluster/#airgapped-installations) mode, the embedded registry will be automatically used to host all application images.
+
+## Image Garbage Collection
+On long running instances, with every application update, new images will be pushed into this registry.
+In order to keep the registry from running out of storage, images that are no longer used will be automatically deleted from the registry.
+
+This feature was introduced in kots 1.48.0 and is currently only supported in embedded kURL clusters when using the Registry add-on.
+Image garbage collection is enabled by default.
+To disable it, execute the following command in the kURL cluster:
+
+```bash
+kubectl patch configmaps kotsadm-confg --type merge -p "{\"data\":{\"enable-image-deletion\":\"false\"}}"
+```
+
+To enable it again, execute the following command:
+```bash
+kubectl patch configmaps kotsadm-confg --type merge -p "{\"data\":{\"enable-image-deletion\":\"false\"}}"
+```
+
+Garbage collection is triggered automatically when a new application version is deployed.
+The [admin-console garbage-collect-images](/kots-cli/admin-console/garbage-collect-images/) command can be used to trigger it manually.
+
+### Restoring deleted images
+Deleted images may be reloaded from application and Admin Console [airgap bundles](/kotsadm/installing/airgap-packages/) using the [admin-console push-images](/kots-cli/admin-console/push-images/) command.
+See [installation instructions](/kotsadm/installing/airgap-packages/#kots-install) for an example of using this command.
+Registry address and namespace can be found on the Registry Settings page in Admin Console.
```

Comment:
I noticed that there isn't a link to another portion of the docs here; having said that, I looked through and there isn't a good doc covering this tab of the admin console. Should I create a ticket & PR for this or do we think it is unneeded? I could likely write this up.

Reply:
Yeah, the docs are definitely missing that.